### PR TITLE
scrcpy 2.0

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  clang -Iapp/scrcpy.p -Iapp -I../app -I../app/src -I/opt/homebrew/Cellar/ffmpeg/5.1.2_6/include -I/opt/homebrew/include -I/opt/homebrew/include/SDL2 -I/opt/homebrew/Cellar/libusb/1.0.26/include/libusb-1.0 -fcolor-diagnostics -DNDEBUG -Wall -Winvalid-pch -Wextra -std=c11 -O3 -D_THREAD_SAFE -MD -MQ app/scrcpy.p/src_screen.c.o -MF app/scrcpy.p/src_screen.c.o.d -o app/scrcpy.p/src_screen.c.o -c ../app/src/screen.c
  ../app/src/screen.c:821:13: error: expected expression
              bool ok = sc_screen_init_size(screen);
              ^
  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/13.0.0/include/stdbool.h:15:14: note: expanded from macro 'bool'
  #define bool _Bool
               ^
  ../app/src/screen.c:822:18: error: use of undeclared identifier 'ok'
              if (!ok) {
                   ^
  2 errors generated.
```

File: Formula/scrcpy.rb [hunk 1]
```diff
@@ -95,3 +98,24 @@ def caveats
...
+             bool ok = sc_screen_init_size(screen);
+             if (!ok) {
+@@ -824,6 +824,7 @@ sc_screen_handle_event(struct sc_screen *screen, SDL_Event *event) {
+                 return false;
+             }
+             return true;
++        }
+         case SC_EVENT_NEW_FRAME: {
+             bool ok = sc_screen_update_frame(screen);
+             if (!ok) {
```
Comment (ZhongRuoyu) [inline Formula/scrcpy.rb:121]: Let's try this. Will upstream if it works.

As for why: https://stackoverflow.com/questions/57742010/error-in-switch-case-a-label-can-only-be-part-of-a-statement-and-a-declaration

Comment (ZhongRuoyu) [reply Formula/scrcpy.rb:121]: It works.

Comment (ZhongRuoyu) [reply Formula/scrcpy.rb]: Fix submitted upstream: https://github.com/Genymobile/scrcpy/pull/3787.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**